images/ose-aws-pod-identity-webhook.yml

```diff
@@ -0,0 +1,23 @@
+container_yaml:
+  go:
+    modules:
+    - module: github.com/openshift/aws-pod-identity-webhook
+content:
+  source:
+    dockerfile: Dockerfile.rhel7
+    git:
+      branch:
+        target: release-{MAJOR}.{MINOR}
+        # This repo does not follow the release- branch convention. Remove fallback when it does.
```

Comment:
i think this can be removed now that the release{4.5|4.6} branches have been created